test/e2e/util/velero/velero_utils.go

```diff
@@ -1290,3 +1297,18 @@ func GetPVB(ctx context.Context, veleroNamespace, namespace string) ([]string, e
 func GetPVR(ctx context.Context, veleroNamespace, namespace string) ([]string, error) {
 	return GetVeleroResource(ctx, veleroNamespace, namespace, "podvolumerestore")
 }
+
+func IsSupportUploaderType(version string) (bool, error) {
+	if strings.Contains(version, "self") {
+		return true, nil
+	}
+	arr := strings.Split(version, ".")
+	if len(arr) >= 3 {
```

Comment:
Maybe we should use [version](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/version#ParseSemantic) lib to compare versions.

[ParseSemantic](https://github.com/kubernetes/apimachinery/blob/v0.26.3/pkg/util/version/version.go#L111)

[AtLeast](https://github.com/kubernetes/apimachinery/blob/v0.26.3/pkg/util/version/version.go#L305)

Reply:
Sure, it's a good choice.